packages/devtools_app/lib/src/common_widgets.dart

```diff
@@ -688,15 +690,15 @@ class AreaPaneHeader extends StatelessWidget implements PreferredSizeWidget {
         alignment: Alignment.centerLeft,
         child: Row(
           children: [
-            Expanded(
-              child: DefaultTextStyle(
-                maxLines: maxLines,
-                overflow: TextOverflow.ellipsis,
-                style: theme.textTheme.subtitle2,
-                child: title,
-              ),
+            DefaultTextStyle(
+              maxLines: maxLines,
+              overflow: TextOverflow.ellipsis,
+              style: theme.textTheme.subtitle2,
+              child: title,
             ),
-            ...actions,
+            ...leftActions,
+            const Spacer(),
```

Comment:
only add spacer if leftActions and rightActions are both not empty.

Reply:
done.